constraints.txt

```diff
@@ -0,0 +1,2 @@
+cloudify-plugins-common==3.4.2
+PyYAML==3.12
```

Comment:
@isaac-s What is the added value of adding the file on the repository level ?,  do not you think it will be easy to give the user who wants to trigger new release the option for specifying these constaintrts as part of build process instead of just add this file on the repository level  ?

Reply:
The only reason why I would have liked to give the user the option to specify constraints during build, is if there was any use of building the same plugin, _at the same version_, using different constraints. But, we never do that and we shouldn't do that. OpenStack plugin version 2.15 will be released with a certain set of constraints only. Once only. So I don't understand why we need to provide constraints to the build process.

Of course, these constraints may change from one plugin version to another. But then, what we do is simply update the `constraints.txt` file in the repo.

Reply:
@isaac-s, I got your point about that

I think in order to apply the changes, we need to update all plugins (aws, openstack, azure, ...) that depend on jenkins build since the process build need to read the  `constraints.txt`  from the repository

Reply:
How do we go about changing Jenkins?

Reply:
@isaac-s I know that I have write permissions to change jobs and scripts. Do you want to take this conversation off slack?

Reply:
@EarthmanT you mean *on* Slack...? sure.